### ISSUE TITLE
Wrong label applied re:Travis by the bot 

### ISSUE THREAD

**Issue by orta** [DefinitelyTyped/dt-mergebot#14]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44263

![Screen Shot 2020-04-28 at 9 35 50 AM](https://user-images.githubusercontent.com/49038/80493634-a7d03900-8933-11ea-924f-90a940e5439f.png)

**Comment by orta** [comment]
Hard to see where this is going wrong so far - 

The JSON is:

```
  "travisResult": "pass",
```

then the logic is:

```
    const failedCI = info.travisResult === TravisResult.Fail;

        if (failedCI) {
            context.labels["The Travis CI build failed"] = true;
            context.responseComments.push(Comments.TravisFailed(info.headCommitAbbrOid, info.author, info.travisUrl!));
        }
```

**Comment by fishcharlie** [comment]
This issue specifically mentions the `The Travis CI build failed`, but I've also noticed some strange behavior with the `Where is Travis?` label as well. Not sure if that is a different issue or the same thing, but just thought I would mention it.

**Comment by andrewbranch** [comment]
Fixed by #20
